modular_citadel/code/game/objects/items/storage/firstaid.dm

```diff
@@ -0,0 +1,140 @@
+	storage_slots = 5
+
+/obj/item/storage/hypospraykit/regular
+	icon_state = "firstaid-mini"
+	desc = "A hypospray kit with general use vials."
+
+/obj/item/storage/hypospraykit/regular/PopulateContents()
+	if(empty)
+		return
+	new /obj/item/reagent_containers/hypospray/mkii/tricord(src)
+	new /obj/item/reagent_containers/glass/bottle/vial/small/preloaded/tricord(src)
+	new /obj/item/reagent_containers/glass/bottle/vial/small/preloaded/tricord(src)
+
+/obj/item/storage/hypospraykit/fire
+	name = "hypospray burn treatment kit"
+	desc = "A specialized hypospray kit for burn treatments. Apply with sass."
+	icon_state = "burn-mini"
+	item_state = "firstaid-ointment"
+
+/obj/item/storage/hypospraykit/fire/PopulateContents()
+	if(empty)
+		return
+	new /obj/item/reagent_containers/hypospray/mkii/burn(src)
+	new /obj/item/reagent_containers/glass/bottle/vial/small/preloaded/kelotane(src)
+	new /obj/item/reagent_containers/glass/bottle/vial/small/preloaded/kelotane(src)
+
+/obj/item/storage/hypospraykit/toxin
+	name = "hypospray toxin treatment kit"
+	icon_state = "toxin-mini"
+	item_state = "firstaid-toxin"
+
+/obj/item/storage/hypospraykit/toxin/PopulateContents()
+	if(empty)
+		return
+	new /obj/item/reagent_containers/hypospray/mkii/toxin(src)
+	new /obj/item/reagent_containers/glass/bottle/vial/small/preloaded/antitoxin(src)
+	new /obj/item/reagent_containers/glass/bottle/vial/small/preloaded/antitoxin(src)
+
+/obj/item/storage/hypospraykit/o2
+	name = "hypospray oxygen deprivation treatment kit"
+	icon_state = "oxy-mini"
+	item_state = "firstaid-o2"
+
+/obj/item/storage/hypospraykit/o2/PopulateContents()
+	if(empty)
+		return
+	new /obj/item/reagent_containers/hypospray/mkii/oxygen(src)
+	new /obj/item/reagent_containers/glass/bottle/vial/small/preloaded/dexalin(src)
+	new /obj/item/reagent_containers/glass/bottle/vial/small/preloaded/dexalin(src)
+
+/obj/item/storage/hypospraykit/brute
+	name = "hypospray brute trauma treatment kit"
+	icon_state = "brute-mini"
+	item_state = "firstaid-brute"
+
+/obj/item/storage/hypospraykit/brute/PopulateContents()
+	if(empty)
+		return
+	new /obj/item/reagent_containers/hypospray/mkii/brute(src)
+	new /obj/item/reagent_containers/glass/bottle/vial/small/preloaded/bicaridine(src)
+	new /obj/item/reagent_containers/glass/bottle/vial/small/preloaded/bicaridine(src)
+
+/obj/item/storage/hypospraykit/tactical
+	name = "hypospray combat medkit"
+	desc = "A hypospray kit best suited for combat situations."
+	icon_state = "tactical-mini"
+	max_w_class = WEIGHT_CLASS_NORMAL
+
+/obj/item/storage/hypospraykit/tactical/PopulateContents()
+	if(empty)
+		return
+	new /obj/item/defibrillator/compact/combat/loaded(src)
+	new /obj/item/reagent_containers/hypospray/mkii/CMO/combat(src)
+	new /obj/item/reagent_containers/glass/bottle/vial/large/preloaded/combat(src)
+	new /obj/item/reagent_containers/glass/bottle/vial/large/preloaded/combat(src)
+
+/obj/item/storage/hypospraykit/cmo
+	name = "CMO Hypospray medkit"
```

Comment:
Would probably be better to break down CMO here as most other head of staff personal items follow that rule.

Reply:
True that